src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFacts.cs

```diff
@@ -406,31 +406,6 @@ internal interface ISyntaxFacts
         bool IsMethodLevelMember([NotNullWhen(true)] SyntaxNode? node);
         bool IsTopLevelNodeWithMembers([NotNullWhen(true)] SyntaxNode? node);
 
-        /// <summary>
-        /// A block that has no semantics other than introducing a new scope. That is only C# BlockSyntax.
-        /// </summary>
-        bool IsScopeBlock([NotNullWhen(true)] SyntaxNode? node);
-
-        /// <summary>
-        /// A node that contains a list of statements. In C#, this is BlockSyntax and SwitchSectionSyntax.
-        /// In VB, this includes all block statements such as a MultiLineIfBlockSyntax.
-        /// </summary>
-        bool IsExecutableBlock([NotNullWhen(true)] SyntaxNode? node);
-        // Violation.  This should return a SyntaxList
-        IReadOnlyList<SyntaxNode> GetExecutableBlockStatements(SyntaxNode? node);
-        // Violation.  This is a feature level API.
-        SyntaxNode? FindInnermostCommonExecutableBlock(IEnumerable<SyntaxNode> nodes);
```

Comment:
ISyntaxFacts defines (in a comment on the type) the sorts of operations it is supposed to provide.  Specifically, it only abstracts over common syntax constructs that both VB and C# have (e.g. an 'if statement').  It is not intended to make many constructs from the same language appear similar.  For that, we want specific other services/features that are based around that concept and are clear on what their particular semantics are.